ClientTests/TelemetryWrapperTests.swift

```diff
@@ -302,4 +302,11 @@ extension XCTestCase {
         XCTAssertTrue(metric.testHasValue(), "Should have value on uuid metric")
         XCTAssertEqual(value, expectedValue, failureMessage)
     }
+
+// MARK: - Migration
+
+    func test_SDWebImageDiskCacheClear_GleanIsCalled() {
+        TelemetryWrapper.recordEvent(category: .information, method: .delete, object: .clearSDWebImageCache)
+        testCounterMetricRecordingSuccess(metric: GleanMetrics.Migration.imageSdCacheCleanup)
+    }
```

Comment:
<img width="900" alt="Screen Shot 2022-06-30 at 4 54 47 PM" src="https://user-images.githubusercontent.com/8919439/176776645-86bd90a5-e761-46a5-acad-3361f2ba8b65.png">

Not sure why I am seeing a discrepancy. My assumption is that setup would clear glean data 🤷🏼 

cc @yoanarios have you encountered anything similar before?

Reply:
I feel one reason this can happen if the app goes in the background when running tests then I can image migration getting called which in turn calls the method to record the event but still what is setup doing if not cleaning glean for testing success 🤔

Reply:
@nbhasin2 no I haven't seen this issue before but I also notice is the first time we use `ttestCounterMetricRecordingSuccess` function wondering if we need to reset something else?